cmd/geth/dbcmd.go

```diff
@@ -175,7 +175,7 @@ WARNING: This is a low-level operation which may cause database corruption!`,
 			utils.RopstenFlag,
 			utils.RinkebyFlag,
 			utils.GoerliFlag,
-			utils.YoloV3Flag,
+			utils.AleutFlag,
```

Comment:
@lightlcient I think you missed a reference to `YoloV3Flag` at https://github.com/quilt/go-ethereum/blob/55d2efc592cdafb0a54cf5c0f51c60f8e911c993/cmd/geth/dbcmd.go#L194

Reply:
@lightclient you can cherry-pick and squash https://github.com/ryanschneider/go-ethereum/commit/4738285e3ca9f4fb4d9d18efd19b097f37c30ea9 into this commit to fix the issue, without this change I get:

```
# github.com/ethereum/go-ethereum/cmd/geth
cmd/geth/dbcmd.go:194:4: undefined: utils.YoloV3Flag
util.go:47: exit status 2
exit status 1
make: *** [geth] Error 1
```